docs/docs/components/maskedtextfield.md

```diff
@@ -0,0 +1,80 @@
+---
+title: MaskedTextField
+---
+
+<DocChip chip='shadow' />
+
+<DocChip chip='name' label="dwc-masked-textfield" />
+
+<JavadocLink type="foundation" location="com/webforj/component/field/MaskedTextField" top='true'/>
+
+The `MaskedTextField` component aims to deliver a configurable and easily validatable text input. It's ideal for apps requiring formatted input, such as financial, e-commerce, and healthcare apps.
+
+
+## Basics
+
+The `MaskedTextField` can be instantiated with or without parameters. You can define an initial value, label, a placeholder text, and a listener in case the value changes.
+
+## Supported masks
```

Comment:
1. Rename the section to **"Mask Rules."**  
2. Include more code snippets showcasing different mask examples, if possible.  
3. Clarify that when a mask consists only of `X`, it's a special case where the field behaves like a standard **TextField**.  
4. At the end, provide a runnable demo featuring a text field with a phone number mask.  
5. Add a note mentioning that patterns can be combined with masks.